archinstall/locales/base.pot

```diff
@@ -46,10 +46,13 @@ msgid "Choose an audio server"
 msgstr ""
 
 msgid ""
-"Only packages such as base, base-devel, linux, linux-firmware, efibootmgr "
+"Only packages such as base, sudo, linux, linux-firmware, efibootmgr "
```

Comment:
Please don't update these translations in here manually as that will mess things up